operations/kubernetes/encryption/README.md

```diff
@@ -231,3 +241,40 @@ If necessary, a forced rewrite of secret data can be performed. Generally unnece
     Waiting for daemon set "cray-k8s-encryption" rollout to finish: 2 of 3 updated pods are available...
     daemon set "cray-k8s-encryption" successfully rolled out
     ```
+
+## Key rotation
+
+Rotation of the encryption keys is similar to disabling encryption. The first cipher passed to `encryption.sh` is the cipher that will be used to encrypt secrets.
+
+All subsequent ciphers provided allow for any secrets encrypted with that cipher to be read but will not be used for new secrets.
+
+* (`ncn-m#`) The `encryption.sh` script needs to be run with the new key as the first switch and the existing key as a subsequent switch.
+
+    As shown in the following command example, always run `encryption.sh` with a leading space on the command line. This will cause Bash to not record the command in the `.bash_history` file.
+
+    ```bash
+     /usr/share/doc/csm/scripts/operations/kubernetes/encryption.sh --enable --aescbc NEWKEYVALUE --aescbc EXISTINGKEYVALUE
+    ```
+
+    Example output:
+
+    ```text
+    ncn-m001 configuration updated ensure all control plane nodes run this same command
+    ```
+
+Once completed, reference Encryption status for details, you may optionally remove the old key from all control plane nodes. Note that removing the old key will mean you will be unable to
+use etcd backups that contain data with that encryption key.
+
+* (`ncn-m#`) The `encryption.sh` script needs to be run again with just the new key.
+
+    As shown in the following command example, always run `encryption.sh` with a leading space on the command line. This will cause Bash to not record the command in the `.bash_history` file.
```

Comment:
```suggestion
    **`NOTE`**: As shown in the following command example, always run `encryption.sh` with a leading space on the command line. This will cause Bash to not record the command in the `.bash_history` file.
```
Same comment as above.